src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(name) {
+    const bittenAnimal = Animal.alive.find(animal => animal === name);
+
+    if (bittenAnimal instanceof Herbivore && bittenAnimal.hidden === false) {
```

Comment:
```suggestion
    if (bittenAnimal instanceof Herbivore && !bittenAnimal.hidden) {
```
You can simplify the condition

Reply:
Did like this at first and than fchanged it and forgot to return it. Thanks for pointing that out